### ISSUE TITLE
Failed to access OTU table to get species pcoa.scores

### ISSUE THREAD

**Issue by altrickter** [joey711/phyloseq#420]
The following warning arises when using plot_heatmap() that may be more general to accessing an OTU table after an ordination is calculated:

```
Warning message:
In scores.pcoa(ps.ord, choices = c(1, 2), display = "species") :
  scores.pcoa: Failed to access OTU table from `physeq` argument, 

              needed for weighted average of OTU/taxa/species points in MDS/PCoA.
```

Here's the code to reproduce this error:

```
data(GlobalPatterns)
# trim
gp <- prune_taxa(taxa_sums(GlobalPatterns) > 1000, GlobalPatterns)
plot_heatmap(gp, method = "PCoA", distance = "bray")
```

Here's my R.Version()

```
$platform
[1] "x86_64-apple-darwin13.4.0"

$arch
[1] "x86_64"

$os
[1] "darwin13.4.0"

$system
[1] "x86_64, darwin13.4.0"

$status
[1] ""

$major
[1] "3"

$minor
[1] "1.2"

$year
[1] "2014"

$month
[1] "10"

$day
[1] "31"

$`svn rev`
[1] "66913"

$language
[1] "R"

$version.string
[1] "R version 3.1.2 (2014-10-31)"

$nickname
[1] "Pumpkin Helmet"
```

And phyloseq version:

```
> packageVersion("phyloseq")
[1] ‘1.10.0’
```

This issue prevents plot_heatmap() from ordering the taxa by the ordination method selected.

**Comment by joey711** [comment]
Yep, in `plot_heatmap` the phyloseq object was simply not being passed along to the `scores` generic. I fixed that in a new branch, and am testing it now before rolling it into the GitHub master. 

If this bug is annoying enough I can see about fixing it in the BioC release version. Let me know.

BTW, I also switched `plot_heatmap` to use ggplot2's newer `geom_raster` function, that should be able to render heatmaps much faster. I'll close this after I can confirm that the tests have all completed without error.

joey
